app/views/legislation/processes/_process.html.erb

```diff
@@ -13,7 +13,7 @@
     </div>
 
     <div class="small-12 medium-11 column end">
-      <%= markdown(process.summary.presence || first_paragraph(process.description)) %>
+      <%= auto_link_already_sanitized_html AdminWYSIWYGSanitizer.new.sanitize((process.summary.presence || first_paragraph(process.description)) %>
```

Comment:
Tests are reporting a syntax error here; it looks like there's an extra parenthesis after `sanitize` :thinking:.

Reply:
It seems that a `)` was missing at the end. 😌